automarker_2/automarker_app/lib/steps.py

```diff
@@ -639,7 +642,12 @@ class PythonDoRequirementsTxtInstall(Step):
     name = "pip install requirements.txt"
 
     def run(self, project_uri, clone_dir_path, self_test, config, fail_fast):
-        command = f"{clone_dir_path/'automarker_venv'/'bin'/'pip'} install -r {clone_dir_path/'requirements.txt'}"
+        if os.name == "nt":
+            pip_executable_path =  'Scripts/pip'
```

Comment:
This doesn't seem DRY or clear. 

Can't we write this down once rather than over and over?  We check this 3 times `if os.name == "nt":`.
And it's really not clear what nt means, if I want to understand this code then I need to google it.